evaluators/rank/ReciprocalRankEvaluator/manifest.yml

```diff
@@ -6,7 +6,7 @@ description: |
 author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
-documentation: https://github.com/jina-ai/jina-hub
+documentation: https://github.com/jina-ai/jina-hub/blob/master/evaluators/rank/NdcgEvaluator/README.md
 version: 0.0.5
```

Comment:
version needs to be updated?

Reply:
good point, can u add the suggestion?

Reply:
@JoanFM I can't because it's not an edited line.

Reply:
done